app/models/application_record.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+
+  def esential
```

Comment:
__[development][Misspell]__

"esential" is a misspelling of "essential"


<!-- Comment by [Sider](https://sider.review) -->